resource_packs/files/unobtrusive/no_fog/fogs/warped_forest_fog_setting.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
this file is doing nothing so this should be deleted

Reply:
no because if a file does not have all the content that mc needs inside of it, it will in this case pull it from default_fogg_setting.json, if you delete the file, it will just use the hardcoded file inside of mcpe
so in this case, it will get the fogs from the default setting json file
the reason, i was able to delete all those other packs, was because they only changed water.

Reply:
> no because if a file does not have all the content that mc needs inside of it, it will in this case pull it from default_fogg_setting.json, if you delete the file, it will just use the hardcoded file inside of mcpe so in this case, it will get the fogs from the default setting json file the reason, i was able to delete all those other packs, was because they only changed water.

ok didn't know that thanks